creepy/_detail/private_key.py

```diff
@@ -23,7 +23,8 @@ def _get_private_key():
 
 
 def load(path: str, passphrase):
-    with creepy.pipe.connect('private_numbers') as session:
+    with creepy.pipe.connect('private_numbers',
+                             hash='5d23c98f71924042eb30dfae4d4a8a7d97edba75c3b9cc98fd9293373833d8dd') as session:
```

Comment:
Does this mean that each time you change `private_numpers.py` you should change this hash?

Reply:
Yes.

Reply:
For debugging you can omit hash.

Reply:
So maybe add such comment to `private_numbers.py` or test that checks hashes.

Reply:
I'm not sure I understand what you mean. I specify hash to make sure I run genuine 'private_numbers', 'private_numbers' itself doesn't know it runs with hash verification or without. Making test that check hashes is pointless as it doesn't make sure 'private_numbers' is genuine.

Reply:
I mean that if I add whitespace to private_number then some other part of code stop working. It's not what you usually expect. User of this lib should know that he shouldn't make such changes. One option is to make test so he can run pytest and see that he shouldn't change it.

Reply:
It's intended documented behaviour. User should not touch `private_numbers` unless he knows what he does. If he changed it by mistake or it was changed by attacker then `connect()` will raise exception telling that script isn't genuine, so it won't run it. Then user should revert changes or notice that script was hacked. If user changed script intentionally, he will copy-paste hash from exception message.